### PR TITLE
Fix Bug 1330134 - Remove all mentions of contributing to Lightbeam

### PR DESCRIPTION
**Pull request by fmarier**
## Description

Remove mentions of a feature that was removed long ago in Lightbeam.

## Bugzilla link

https://bugzilla.mozilla.org/show_bug.cgi?id=1330134

## Testing

I manually removed the same HTML nodes on production using the devtools.

## Checklist
- [x] Requires l10n changes.
- [ ] Related functional & integration tests passing.

### CODE REVIEW

File: bedrock/lightbeam/templates/lightbeam/lightbeam.html [hunk 1]
```diff
@@ -107,15 +101,6 @@ <h2 class="page-slogan">{{_('How Lightbeam Works')}}</h2>
       </p>
     </figcaption>
   </figure>
-  <figure class="span4 contribute">
-    <img src="{{ static('img/lightbeam/Lightbeam_Illustration3.png') }}" alt="">
```
Comment (alexgibson) [inline bedrock/lightbeam/templates/lightbeam/lightbeam.html]: Can you please remove this image from the repo if it's no longer being used? Thanks

File: bedrock/lightbeam/templates/lightbeam/lightbeam.html [hunk 2]
```diff
@@ -107,15 +101,6 @@ <h2 class="page-slogan">{{_('How Lightbeam Works')}}</h2>
       </p>
     </figcaption>
   </figure>
-  <figure class="span4 contribute">
```
Comment (alexgibson) [inline bedrock/lightbeam/templates/lightbeam/lightbeam.html]: Removing this from the page kinda leaves things off balance visually. Can I suggest you change the two remaining items to have a class of `span6` so they make better use of the available space?

File: bedrock/lightbeam/templates/lightbeam/lightbeam.html [hunk 3]
```diff
@@ -80,12 +80,6 @@ <h2 class="page-slogan">{{_('How Lightbeam Works')}}</h2>
       {{_('Not all tracking is bad. Many services rely on user data to provide relevant content and enhance your online experience.')}} {{_('But tracking can happen without the user’s knowledge.')}} {{_('That’s not okay for some.')}} {{_('It should be you who decides when, how and if you want your browsing data to be shared. We recognize the importance of transparency and our mission is all about empowering users — both with tools and information.')}}  <a href="{{ url('lightbeam.about') }}"> {{_('About Lightbeam')}} »</a>
     </p>
   </div>
-  <div class="span6">
```
Comment (alexgibson) [inline bedrock/lightbeam/templates/lightbeam/lightbeam.html]: Changing the "Tracking & Privacy" block above to `span8` would also help pad things out a little more visually.

**Comment by fmarier** [comment]
@alexgibson I've addressed your review comments.

**Pull request merged**